ee-galleon-pack/src/main/resources/layers/standalone/batch-jberet/layer-spec.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="batch-jberet">
+    <dependencies>
+        <!-- CDI should be optional but to be conservative in the first cut at this, require it
+        <layer name="cdi" optional="true"/>
+        -->
+        <layer name="cdi"/>
+        <layer name="ee"/>
+        <layer name="elytron"/>
+        <layer name="transactions"/>
+    </dependencies>
+    <feature-group name="batch-jberet"/>
```

Comment:
@bstansberry , jberet feature-group doesn't secure jberet with elytron security-domain. In The example CLI script we are defining it is. I am wandering if we should secure jberet in the script: https://github.com/wildfly/wildfly-proposals/pull/294#discussion_r381855034

Reply:
Just to close out this conversation -- the discussion on wildfly-proposals says that adding that bit of config is correct when elytron is used. Which raises the question of how this layer should deal with that. I posted a separate general comment raising that question.